packages/migration-sdk-viem/src/fetchers/index.ts

```diff
@@ -0,0 +1,53 @@
+import type { Address } from "@morpho-org/blue-sdk";
+
+import type { MigratablePosition } from "../positions/index.js";
+import { MigratableProtocol } from "../types/index.js";
+
+import type { FetchParameters } from "@morpho-org/blue-sdk-viem";
+import { fromEntries } from "@morpho-org/morpho-ts";
+import type { Client } from "viem";
+import { fetchAaveV2Positions } from "./aaveV2/aaveV2.fetchers.js";
+import { fetchAaveV3Positions } from "./aaveV3/aaveV3.fetchers.js";
+import { fetchAaveV3OptimizerPositions } from "./aaveV3Optimizer/aaveV3Optimizer.fetchers.js";
+import { fetchCompoundV2Positions } from "./compoundV2/compoundV2.fetchers.js";
+import { fetchCompoundV3Positions } from "./compoundV3/compoundV3.fetchers.js";
```

Comment:
Maybe also export them each indivdually?

Reply:
I didn't want to "polute" the exports with `n` functions while calling  `fetchAaveV2Positions` is equivalent to `fetchMigratablePositions(..., {protocols: [MigratableProtocol.aaveV2]})`

Reply:
not equivalent in types!